### DISCUSSION TITLE
Chat assistant now working

### DISCUSSION THREAD

**Discussion by ashishjaswal2002** [zed-industries/zed#8191] [Feature Requests]
After adding open api key. My chat assistant is not working.

**Comment by Moshyfawn** [comment]
This isn't nearly enough information to go off of. It is recommended that you check your OpenAI credits, subscription or token expiration dates, and ensure that you are using the correct key.

The feature should work seamlessly with a valid OpenAI key at this time.

**Reply by SMak06** [reply]
How do I change or remove the API key once it has been set?

**Reply by Moshyfawn** [reply]
> How do I change or remove the API key once it has been set?

#8200

**Comment by xpe** [comment]
@ashishjaswal2002 Ready to close this discussion?
